x/interchainstaking/keeper/keeper.go

```diff
@@ -537,6 +522,28 @@ func (k *Keeper) EmitDepositIntervalQuery(ctx sdk.Context, zone *types.Zone) {
 	)
 }
 
+func (k *Keeper) QueryValidatorSigningInfo(ctx sdk.Context, connectionID, chainID string, validator stakingtypes.Validator) error {
+	queryType := fmt.Sprintf("/cosmos/slashing/v1beta1/signing_infos/%s", validator.GetOperator().String())
```

Comment:
I believe this param is consensus are not operator address iirc

Reply:
And instead of hitting the SigningInfosQuery endpoint, we want to query the kv entry directly, with key add generated here: https://github.com/cosmos/cosmos-sdk/blob/d6d929843bbd331b885467475bcb3050788e30ca/x/slashing/keeper/signing_info.go#L14

Reply:
Yes, I forgot to update it, thank @joe-bowman